packages/@core.keystone/test.utils.js

```diff
@@ -368,6 +368,11 @@ const isMongo = () => {
     return conf.DATABASE_URL.startsWith('mongo')
 }
 
+const getRandomBetween = (min, max) => min + Math.floor(Math.random() * (max - min))
+
+const getRandomItem = list => list[getRandomBetween(0, list.length - 1)]
```

Comment:
it's better to inline such things. But ok if you want you can put it here. As for me, It's not so helpful

Reply:
it's shorter and cleaner actually